src/pages/Main/PersonProfile/index.tsx

```diff
@@ -46,17 +51,19 @@ const useStyles = makeStyles({
 const PersonProfile = React.memo(function () {
     const personID = usePersonID();
     const isBigScreen = useBigScreen();
-    const [persons, setPersons] = useState({});
-    const [fallbackPerson, setFallbackPerson] = useState(null);
+    const [persons, setPersons] = useState<PersonsDict>({});
+    const [fallbackPerson, setFallbackPerson] =
+        useState<ReturnType<typeof parseResponse>>(null);
     const serverFetch = useCancellableFetch();
-    const person = useMemo(() => persons[personID], [persons, personID]);
+    const person = useMemo(() => persons[personID ?? 0], [persons, personID]);
```

Comment:
personID is int, why null-coalescing it?

Reply:
use perseon id can return undefined see the new version